src/pages/ConsumerEServiceCatalogPage/components/CatalogCard.tsx

```diff
@@ -31,16 +35,48 @@ export const CatalogCard: React.FC<CatalogCardProps> = ({ eservice }) => {
     )
   }
 
+  const sxCardProps: SxProps =
+    disabled === true
+      ? {
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          minHeight: 410,
+          opacity: 0.5,
+        }
+      : {
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          minHeight: 410,
+        }
+
+  const linkEservice = (
+    <>
+      <Tooltip key="subscribe-tooltip" title={disabled ? t('subscribeTooltip') : ''} arrow>
+        <span>
+          <Link
+            as="button"
+            size="small"
+            variant="contained"
+            to="SUBSCRIBE_CATALOG_VIEW"
+            params={{
+              eserviceId: eservice.id,
+              descriptorId: eservice.activeDescriptor?.id ?? '',
+            }}
+            onFocusVisible={handlePrefetch}
+            color="primary"
+            disabled={disabled}
+          >
+            {tCommon('actions.inspect')}
+          </Link>
+        </span>
+      </Tooltip>
+    </>
+  )
```

Comment:
Why did you extract it in a separate constant?
The React fragments and the key prop are not needed.
To make the Tooltip "optional" you need to pass the open prop and optionally pass `false` or `undefined`, you can see an example here `/src/components/shared/StepActions.tsx`.
You can also remove the ternary on the title prop